common/djangoapps/student/management/commands/transfer_students.py

```diff
@@ -52,7 +52,6 @@ class Command(TrackedCommand):
                     help="If True, try to transfer certificate items to the new course.")
     )
 
-    @transaction.commit_manually
```

Comment:
@symbolist @doctoryes since the transaction isn't in a try block. Does that mean that if command fails on a user part way through the transaction the database would end up in an inconsistent state as there is nothing to catch the exception? You would end up with all the students to that point transferred but the remainder not transferred. 

Also would this end up failing on the rerun as the student has already been transferred to the new course?

We have two routes of action here:
1) add the try/catch and migrate students with no errors and report the ones that were not moved 
or
2) fail the entire transaction and keep all students in a consistent state.

I think that #2 is better as it would then allow us to troubleshoot and fix the issue without any inconsistencies in the data.